debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -789,6 +789,16 @@ public static EventDeserializationFailureHandlingMode parse(String value) {
                              + "'warn' the problematic event and its binlog position will be logged and the event will be skipped;"
                              + "'ignore' the problematic event will be skipped.");
 
+    public static final Field SNAPSHOT_SELECT_STATEMENT_OVERRIDES_BY_TABLE = Field.create("snapshot.select.statement.overrides")
```

Comment:
What's the rationale for having this additional property besides the props with the actual select statements? Wouldn't those be sufficient to control this functionality?

Reply:
Ok, realizing that @jpechane had been requesting this. Works for me :)